include/dav1d/picture.rs

```diff
@@ -189,7 +192,7 @@ impl Rav1dPictureDataComponentInner {
     /// so it is sound to further subdivide it into disjoint `&mut`s.
     pub fn wrap_buf<BD: BitDepth>(buf: &mut [BD::Pixel], stride: usize) -> Self {
         let buf = AsBytes::as_bytes_mut(buf);
-        let ptr = NonNull::new(buf.as_mut_ptr()).unwrap();
+        let ptr = Unique::from_ref_mut(buf).cast();
```

Comment:
How is this unique in the wrap_buf case? We're borrowing a reference to this data, right? So should we really use unique here?

Reply:
A `&mut` reference is always unique.  So it can be `&mut` or owned.  `core::ptr::Unique` `impl`s `From<&mut T>` even.